### PR TITLE
Use 'namespace' instead of 'module' in the 'Declaration Merging' section

### PR DESCRIPTION
**Pull request by DanielRosenwasser**

### CODE REVIEW

File: pages/Declaration Merging.md [hunk 1]
```diff
@@ -13,14 +13,14 @@ Declaration merging is not limited to just two declarations, as any number of de
 
 # Basic Concepts
 
-In TypeScript, a declaration exists in one of three groups: namespace/module, type, or value.
-Declarations that create a namespace/module are accessed using a dotted notation when writing a type.
+In TypeScript, a declaration exists in one of three groups: namespace, type, or value.
+Declarations that create a namespace are accessed using a dotted notation when writing a type.
 Declarations that create a type do just that, create a type that is visible with the declared shape and bound to the given name.
 Lastly, declarations create a value are those that are visible in the output JavaScript (e.g. functions and variables).
```
Comment (sandersn) [inline pages/Declaration Merging.md]: While you're here, can you add the missing 'that' in "declarations that create a value"?

**Comment by sandersn** [comment]
Never mind, I think the entire paragraph needs a rewrite. I'll send a PR shortly that does that. (Based on my understanding of modules -- it'll probably still need some revision.)

**Comment by DanielRosenwasser** [comment]
Any other feedback? @RyanCavanaugh @danquirk @mhegazy?

**Comment by mhegazy** [comment]
:+1:

**Comment by sandersn** [comment]
:+1:

**Pull request merged**